### ISSUE TITLE
Missing field `paymentId`

### ISSUE THREAD

**Issue by tworec** [golemfactory/yagna#1369]
@blushade is reporting, using `v0.7.0-rc4`

```
[2021-05-27T13:32:55Z ERROR ya_payment::processor] Error sending payment message to provider: Err(RemoteError("/net/0x51b9e92efe08b51b6b1ca56973e7426060c89384/payment/SendPayment", "GSB failure: Bad content: Serde Error: missing field `paymentId`"))
```

`0x51b9e92efe08b51b6b1ca56973e7426060c89384` is a `witek.b1` address and it is running `v0.6.7-beta.1`

**Comment by nieznanysprawiciel** [comment]
Related to: https://github.com/golemfactory/yagna/issues/1360

**Comment by maaktweluit** [comment]
Resolved by #1370
